### PR TITLE
Revert "fix vomiting not stunning"

### PR DESCRIPTION
**Pull request by Iglpigl**
Reverts discordia-space/CEV-Eris#7798

### CODE REVIEW

**Comment by ValoTheValo** [comment]
clearly the solution to stun timers being broken is to add even more stun timers to the game

**Comment by ValoTheValo** [comment]
merge this maintainers

File: code/modules/mob/living/carbon/human/human.dm [hunk 1]
```diff
@@ -693,7 +693,7 @@ var/list/rank_prefix = list(\
 			sleep(150)	//15 seconds until second warning
 			to_chat(src, SPAN_WARNING("You feel like you are about to throw up!"))
 			sleep(100)	//and you have 10 more for mad dash to the bucket
-		Stun(3)
+		Stun(2)
```
Comment (Firefox13) [inline code/modules/mob/living/carbon/human/human.dm:696]:
```suggestion

```
Consider the following:

**Review by TheShown911 — APPROVED** [review]
Zased.

**Pull request merged**